<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Pokemon by name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Pokemon by ID:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="typeScrollArea">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>132</width>
       <height>444</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="normalCheckBox">
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fireCheckBox">
        <property name="text">
         <string>Fire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="waterCheckBox">
        <property name="text">
         <string>Water</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="grassCheckBox">
        <property name="text">
         <string>Grass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="electricCheckBox">
        <property name="text">
         <string>Electric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="iceCheckBox">
        <property name="text">
         <string>Ice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fightingCheckBox">
        <property name="text">
         <string>Fighting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="poisonCheckBox">
        <property name="text">
         <string>Poison</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="groundCheckBox">
        <property name="text">
         <string>Ground</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="flyingCheckBox">
        <property name="text">
         <string>Flying</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="psychicCheckBox">
        <property name="text">
         <string>Psychic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bugCheckBox">
        <property name="text">
         <string>Bug</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rockCheckBox">
        <property name="text">
         <string>Rock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ghostCheckBox">
        <property name="text">
         <string>Ghost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="darkCheckBox">
        <property name="text">
         <string>Dark</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dragonCheckBox">
        <property name="text">
         <string>Dragon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="steelCheckBox">
        <property name="text">
         <string>Steel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fairyCheckBox">
        <property name="text">
         <string>Fairy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="regionScrollArea">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-3</y>
       <width>142</width>
       <height>228</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="kantoCheckBox">
        <property name="text">
         <string>Kanto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="johtoCheckBox">
        <property name="text">
         <string>Johto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hoennCheckBox">
        <property name="text">
         <string>Hoenn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sinnohCheckBox">
        <property name="text">
         <string>Sinnoh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unovaCheckBox">
        <property name="text">
         <string>Unova</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kalosCheckBox">
        <property name="text">
         <string>Kalos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alolaCheckBox">
        <property name="text">
         <string>Alola</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="galarCheckBox">
        <property name="text">
         <string>Galar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hisuiCheckBox">
        <property name="text">
         <string>Hisui</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by Pokemon type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by Pokemon Region:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="statScrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>412</width>
       <height>162</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="attackMinLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="hpMaxLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hpMinLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="attackMaxLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="defenseCheckBox">
        <property name="text">
         <string>Defense</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="attackCheckBox">
        <property name="text">
         <string>Attack</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="specialDefenseCheckBox">
        <property name="text">
         <string>Special Defense</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="specialAttackCheckBox">
        <property name="text">
         <string>Special Attack</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="hpCheckBox">
        <property name="text">
         <string>HP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="speedCheckBox">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="defenseMinLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="specialAttackMinLineEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="specialDefenseMinLineEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="speedMinLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="defenseMaxLineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="specialAttackMaxLineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="specialDefenseMaxLineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="speedMaxLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="nameSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by Stat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="searchResultsTableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>581</width>
      <height>241</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="type2LineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="evolvesFromPictureLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>380</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Pokemon Picture here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>360</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evolved from:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="regionLineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createPokemonButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="region2LineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePokemonButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="type1LineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>141</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Attack:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Special Defense:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>HP:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Special Attack:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Defense:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hpLabel"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="attackLabel"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="defenseLabel"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="specialAttackLabel"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="specialDefenseLabel"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="speedLabel"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="totalLabel"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="weightLabel"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="idLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>360</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evolves to:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region(s) found:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pokemonNameLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pokemonPictureLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Pokemon Picture here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="evolvesToPictureLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>380</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Pokemon Picture here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evolution ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evolving ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="evolutionIDLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="evolvingIDLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
